src/scenes/EsiJaPerusopetus/lomake/2-OpetustaAntavatKunnat.js

```diff
@@ -334,14 +334,13 @@ const OpetustaAntavatKunnat = React.memo(props => {
           localizations: {
             accept: intl.formatMessage(common.accept),
             areaOfActionIsUndefined: intl.formatMessage(
-              wizard.noMunicipalitiesSelected
+              wizard.areaOfActionIsUndefined
```

Comment:
Tässä sama juttu. Muutos pitää peruuttaa.

Reply:
Näin on.